src/Engine/Path.java

```diff
@@ -157,6 +160,17 @@ public int getPathLength()
     return waypoints.size();
   }
 
+  public int getFuelCost(UnitModel model, GameMap map)
+  {
+    int cost = 0;
+    for (int i = 1; i < waypoints.size(); i++)
```

Comment:
If you are gonna do something sneaky like iterate from 1, you should leave a comment to call it out and explain why, to keep future developers from trying to "fix" it.